developer/build_plugin.py

```diff
@@ -517,6 +552,15 @@ def main():
             --bootstrap-core-uri='sgtk:descriptor:dev?path=~/dev/tk-core'
             ~/dev/tk-maya/plugins/basic /tmp/maya-plugin
 
+By using the '--bake' option, you can build a plugin with an immutable configuration
+where every single Toolkit component is cached and frozen to the version retrieved at
```

Comment:
i think we should mention that in this case the `info.yml` file is not part of the plugin build.